ports/powerpc/Makefile

```diff
@@ -0,0 +1,70 @@
+include ../../py/mkenv.mk
+
+CROSS = 0
```

Comment:
It looks like this variable is unused...?

Reply:
Removing